src/Resources/ResourceManager/Implementation/DeploymentStacks/RemoveAzManagmentGroupDeploymentStack.cs

```diff
@@ -47,6 +49,11 @@ public class RemoveAzManagementGroupDeploymentStack : DeploymentStacksCmdletBase
         [ValidateNotNullOrEmpty]
         public string ManagementGroupId { get; set; }
 
+        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true, ParameterSetName = RemoveByStackObjectParameterSetName,
+            HelpMessage = "The stack PS object")]
+        [ValidateNotNullOrEmpty]
+        public PSDeploymentStack StackObjet { get; set; }
```

Comment:
StackObject is a complex object. Please add a model cmdlet for creating it. Refer to an [example](https://github.com/Azure/azure-powershell/blob/main/src/KeyVault/KeyVault/Commands/VirtualNetworkRuleSet/NewAzureKeyVaultNetworkRuleSetObject.cs). And add examples for it.

Reply:
In that case, the parameter name should be "InputObject".  Please refer to [Remove-* cmdlet example](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/examples/remove-cmdlet-example.md).